core/API/DataTablePostProcessor.php

```diff
@@ -453,6 +455,17 @@ public function computeProcessedMetrics(DataTable $dataTable)
                 $computedValue = $processedMetric->compute($row);
                 if ($computedValue !== false) {
                     $row->addColumn($name, $computedValue);
+
+                    // Add a trend column for evolution metrics
+                    if ($processedMetric instanceof EvolutionMetric) {
+                        $isLowerBetter = Metrics::isLowerValueBetter($name);
```

Comment:
This is actually not fully correct. `$name` holds the name of the evolution metric, but `Metrics::isLowerValueBetter` expects the column metric, which is the wrapped metric of the evolution metric. I will push a fix, so you can have a quick look and merge if it's fine

Reply:
Thanks, that makes sense.